### DISCUSSION TITLE
axios is always null

### DISCUSSION THREAD

**Discussion by justageek** [usebruno/bruno#1634] [General]
### I am trying to write a pre request script for my collection but axios is always null when I try
```
const axios = require('axios');

**Comment by justageek** [comment]
I have added a package.json file and installed axios in the root of my collection still nothing.

**Reply by dycor** [reply]
Someone found a solution ?

**Reply by hotbinz** [reply]
Someone found a solution ?

**Comment by hotbinz** [comment]
I finally discovered the problem，
Bruno scripts Using JS pre compilation, so not object,
```const axios = require('axios');

console.info(axios)   //is null

console.info(axios.toString())   //not null
```
